composer.json

```diff
@@ -42,16 +42,16 @@
             "type": "package",
             "package": {
                 "name": "sass/sass-spec",
-                "version": "3.6.0",
+                "version": "2020.08.10",
```

Comment:
As sass-spec does not tag releases anymore, considering it a living spec, I decided to use date-based versions here. We don't really care about what the version is, as long as it is the same in `require-dev`. But it is important that this version gets changed when we change the referenced commit, so that `composer update` knows that this is a different version. Using the date is the easy way here (and compatible with composer) and gives an info about how uptodate the sass-spec is.